app/workers/migrate_data_to_work_field_worker.rb

```diff
@@ -0,0 +1,13 @@
+# NOTE: we can remove this once the DUS using it finished
+class MigrateDataToWorkFieldWorker
+  include Sidekiq::Worker
+
+  def perform(profile_id)
+    profile = Profile.find_by(id: profile_id)
```

Comment:
Not my favorite pattern but since we use it pretty consistently across our workers I decided to use it here too...